cards/faq-accordion/component.js

```diff
@@ -76,18 +76,24 @@ class faq_accordionCardComponent extends BaseCard['faq-accordion'] {
 
     if (stayExpanded && this.getState('feedbackSubmitted')) {
       isExpanded = true;
-      cardEl.classList.toggle(accordionExpandedClass, isExpanded);
+      if (!cardEl.classList.contains(accordionExpandedClass)) {
+        cardEl.classList.toggle(accordionExpandedClass, isExpanded);
+      }
       accordionToggleEl.setAttribute('aria-expanded', isExpanded ? 'true' : 'false');
     }
-    cardEl.classList.toggle(stayExpandedClass, false);
+    if (cardEl.classList.contains(stayExpandedClass)) {
```

Comment:
this always get remove for the next re-render right? I think you can just do `cardEl.classList.remove(stayExpandedClass); ` without if statement